operator/v1/types_dns.go

```diff
@@ -287,6 +287,17 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+	// protocolPolicy configures the protocol policy for upstream DNS requests.
+	//
+	// This field allows a cluster admin to optionally configure:
+	// * None configures CoreDNS for to use its default upstream protocol selection.
+	// * ForceTCP configures CoreDNS to use TCP for all upstream DNS requests, even if the client request uses UDP.
+	//
+	// Note that this does not affect the protocol of DNS requests made by clients internal to the cluster.
```

Comment:
This should be moved up to the first line of the comment if it applies to any selection.   Otherwise, change "this" to the selection you're referencing.

Reply:
Okay - substituted the field name.